test-framework/junit5/src/main/java/io/quarkus/test/junit/QuarkusTestExtension.java

```diff
@@ -1089,13 +1093,16 @@ private Object runExtensionMethod(ReflectiveInvocationContext<Method> invocation
         }
     }
 
-    private Method determineTCCLExtensionMethod(ReflectiveInvocationContext<Method> invocationContext, Class<?> c)
+    private Method determineTCCLExtensionMethod(Method orginalMethod, Class<?> c)
```

Comment:
small typo, should be `originalMethod` 😉

Reply:
🤦‍♂️, thanks. I'll wait until others have reviewed to address everything in one go.